src/eddymotion/estimator.py

```diff
@@ -150,9 +151,7 @@ def fit(
                         pbar.set_description_str(
                             f"Pass {i_iter + 1}/{n_iter} | Fit and predict b-index <{i}>"
                         )
-                        data_train, data_test = dwdata.logo_split(i, with_b0=True)
-                        grad_str = f"{i}, {data_test[1][:3]}, b={int(data_test[1][3])}"
-                        pbar.set_description_str(f"[{grad_str}], {n_jobs} jobs")
+                        data_train, data_test = logo_split(dwdata, i)
```

Comment:
Accidentaly removed these two lines, adding back:
```suggestion
                        data_train, data_test = logo_split(dwdata, i)
                        grad_str = f"{i}, {data_test[1][:3]}, b={int(data_test[1][3])}"
                        pbar.set_description_str(f"[{grad_str}], {n_jobs} jobs")
```